tardis/io/config_reader.py

```diff
@@ -1007,6 +1007,10 @@ def from_config_dict(cls, config_dict, atom_data=None, test_parser=False,
         ##### Monte Carlo Section
 
         montecarlo_section = validated_config_dict['montecarlo']
+        montecarlo_section['no_of_packets'] = \
+            int(montecarlo_section['no_of_packets'])
+        montecarlo_section['last_no_of_packets'] = \
+            int(montecarlo_section['last_no_of_packets'])
```

Comment:
I think there will be several small issues like this which can be 'fixed' during the refactor. Should I make one Issue for all of them?